packages/docs/src/pages/index.js

```diff
@@ -9,55 +9,292 @@ import { Hero } from '@site/src/components/Hero'
+                        ...(style ?? {}),
+                    }}
+                >
+                    {children}
+                </div>
+            </div>
+        </a>
+    )
+}
+
+const Section = ({ title, description }) => (
+    <div
+        style={{
+            display: 'flex',
+            flexDirection: 'column',
+            borderTop: '1px solid black',
+            width: '250px',
+            marginTop: '56px',
+            color: 'black',
+        }}
+    >
+        <div style={{ height: '24px' }} />
+        <div style={{ fontSize: '20px', fontWeight: 700, lineHeight: '28px' }}>
+            {title}
+        </div>
+        <div style={{ height: '24px' }} />
+        <div className={styles.paragraph}>{description}</div>
+    </div>
+)
 
 function HomepageHeader() {
     const { siteConfig } = useDocusaurusContext()
     return (
-        <header className={clsx('hero hero--primary', styles.heroBanner)}>
+        <div
+            style={{
+                maxWidth: '1196px',
+                margin: 'auto',
+                paddingLeft: '16px',
+                paddingRight: '16px',
+                paddingTop: '104px',
+                display: 'flex',
+                flexDirection: 'column',
+            }}
+        >
+            <div
+                style={{
+                    fontSize: '96px',
+                    fontWeight: 700,
+                    lineHeight: '100%',
+                    alignSelf: 'center',
+                }}
+            >
+                A protocol built for handling user data anonymously.
+            </div>
+            <div style={{ height: '24px' }} />
+            <div className={styles.paragraph} style={{ maxWidth: '768px' }}>
+                UniRep is a Zero-Knowledge Protocol for user data & reputation
+                management. We use pioneering technology to offer a space for
+                developers and users alike to explore the potential of
+                privacy-centered online interactions.
+            </div>
+            <div style={{ height: '24px' }} />
+            <div style={{ alignSelf: 'flex-start' }}>
+                <Button href="/docs/welcome">Start building</Button>
+            </div>
+            <div style={{ height: '124px' }} />
+            <div
+                style={{
+                    background:
+                        'linear-gradient(135.45deg, #FEE4CB 0%, #DBF2F2 95.26%)',
+                    borderRadius: '32px',
+                    padding: '104px 113px 104px 113px',
+                    display: 'flex',
+                    flexDirection: 'column',
+                }}
+            >
+                <div
+                    style={{
+                        color: 'black',
+                        fontSize: '56px',
+                        fontWeight: 700,
+                        lineHeight: '53px',
+                    }}
+                >
+                    Use UniRep to build a Reddit clone
```

Comment:
@ChialiT 
I think only mentioning **Reddit** is too limited

Reply:
this is not the intention :)